exec/cnex/nstring.c

```diff
@@ -188,6 +184,12 @@ char *tprintf(char *dest, TString *s)
     return dest;
 }
 
+char *to_string(char *dest, uint32_t val)
+{
+    snprintf(dest, TSTR_N, "%d", val);
```

Comment:
But this function is never called.  It's only called via the macro.  Should this entire function be wrapped in the macro instead?  Actually, this function should be a static function, because it's not available to anyone, outside of the macro.